packages/atomic/cypress/e2e/search-box/instant-results/search-box-instant-results.cypress.ts

```diff
@@ -161,52 +161,53 @@ describe('Instant Results Test Suites', () => {
       SearchBoxAssertions.assertSuggestionIsHighlighted(1);
     });
     describe('when navigating back from result to query', () => {
-      before(() => {
+      beforeEach(() => {
         setupWithSuggestionsAndRecentQueries();
-        SearchBoxSelectors.inputBox().type(
-          `${downKeys(2)}{rightarrow}{leftarrow}`,
-          delay()
+        SearchBoxSelectors.inputBox().focus();
+        InstantResultsSelectors.results().should(
+          'have.length',
+          numOfInstantResults
         );
-      });
-      after(() => {
-        SearchBoxSelectors.inputBox().clear({force: true});
-      });
-      InstantResultsAssertions.assertNoResultIsSelected();
-      SearchBoxAssertions.assertSuggestionIsSelected(0);
-      SearchBoxAssertions.assertHasText('Recent query 0');
-    });
-    describe('when navigating back from result to query', () => {
-      before(() => {
-        setupWithSuggestionsAndRecentQueries();
         SearchBoxSelectors.inputBox().type(
           `${downKeys(2)}{rightarrow}{leftarrow}`,
           delay()
         );
       });
-      after(() => {
+      afterEach(() => {
         SearchBoxSelectors.inputBox().clear({force: true});
       });
       InstantResultsAssertions.assertNoResultIsSelected();
       SearchBoxAssertions.assertSuggestionIsSelected(0);
       SearchBoxAssertions.assertHasText('Recent query 0');
     });
     describe('when pressing enter on a result', () => {
-      before(() => {
+      beforeEach(() => {
         setupWithSuggestionsAndRecentQueries();
+        SearchBoxSelectors.inputBox().focus();
+        InstantResultsSelectors.results().should(
+          'have.length',
+          numOfInstantResults
+        );
```

Comment:
We focus, and then we wait for things to be actually there before interacting. That is what a user would do.